src/applications/disability-benefits/all-claims/content/toxicExposure.jsx

```diff
@@ -312,10 +312,15 @@ export function validateSelections(
 ) {
   const { [objectName]: items = {} } = formData?.toxicExposure;
 
+  const selectedLocations = Object.values(items).filter(value => value === true)
+    .length;
+
   if (
     items?.none === true &&
-    (Object.values(items).filter(value => value === true).length > 1 ||
-      !!getOtherFieldDescription(formData, otherObjectName))
+    (selectedLocations > 1 ||
+      !!getOtherFieldDescription(formData, otherObjectName)) &&
+    // If user selects `none` and `not sure`, we don't want to show the error
+    (selectedLocations > 2 || items?.notsure !== true)
```

Comment:
Selecting 'none' and entering an 'other' value should result in a validation error but the error is no longer displaying and it incorrectly lets me continue to the next page
![image](https://github.com/department-of-veterans-affairs/vets-website/assets/127446042/abc7170c-7e9a-48c1-8ad4-19d45b7d3562)